src/chia_plot.cpp

```diff
@@ -9,6 +9,7 @@
 #include <chia/phase2.hpp>
 #include <chia/phase3.hpp>
 #include <chia/phase4.hpp>
+#include <chia/chia_filesystem.hpp>
```

Comment:
You'll have to commit that header too.

Reply:
lol good timing

Reply:
I will try to add whatever i can for now and i am running some parallel ploting tests to draw some conclussions. 
The final dir parameter if you don't mind even if it is not used now will be prepared for input and some sanity checks for farmer and pool keys since if you don't provide valid values you have something like: 
terminate called after throwing an instance of 'std::invalid_argument'
  what():  G1Element::FromBytes: Invalid size. 

Will try to stick in the "chia" paradigm and re-use whatever i can.  I know it's not a big help.

Reply:
I'll add cxxopt command line parsing later, together with final dir copy.